### PR TITLE
Update the agents page with new UX

### PR DESCRIPTION
**Pull request by sabaimran**
- Use icons/colors for setting the styling of agents
- Update automations page to use the shadcn cards: https://github.com/shadcn-ui/ui
Screenshot 2024-07-14 at 01 00 00

### CODE REVIEW

**Review by debanjum — APPROVED** [review]
Looks good! Glad the speed of iteration on the UX feels faster now

File: src/interface/web/app/automations/layout.tsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import NavMenu from '../components/navMenu/navMenu';
 import styles from './automationsLayout.module.css';
 import { Toaster } from "@/components/ui/toaster";
 
+import "../globals.css";
+
```
[the remark this answers is not in the document]
Comment (sabaimran) [reply src/interface/web/app/automations/layout.tsx:8]: Yes, all the pages need globals.css.

File: src/interface/web/app/agents/page.tsx [hunk 2]
```diff
@@ -39,139 +82,216 @@ async function openChat(slug: string, userData: UserProfile | null) {
...
+    if (color === 'teal') return `bg-teal-500 hover:bg-teal-600`;
+    if (color === 'cyan') return `bg-cyan-500 hover:bg-cyan-600`;
+    if (color === 'lime') return `bg-lime-500 hover:bg-lime-600`;
+    if (color === 'indigo') return `bg-indigo-500 hover:bg-indigo-600`;
+    if (color === 'fuschia') return `bg-fuschia-500 hover:bg-fuschia-600`;
+    if (color === 'rose') return `bg-rose-500 hover:bg-rose-600`;
+    if (color === 'sky') return `bg-sky-500 hover:bg-sky-600`;
+    if (color === 'amber') return `bg-amber-500 hover:bg-amber-600`;
+    if (color === 'emerald') return `bg-emerald-500 hover:bg-emerald-600`;
+    return `bg-gray-500 hover:bg-gray-600`;
```
[the remark this answers is not in the document]
Comment (sabaimran) [reply src/interface/web/app/agents/page.tsx:122]: No, this is what I had originally. Tailwind won't accept computed values, they have to be full strings. https://stackoverflow.com/questions/69687530/dynamically-build-classnames-in-tailwindcss

[the remark this answers is not in the document]
Comment (sabaimran) [reply src/interface/web/app/agents/page.tsx:122]: Same for below.

**Pull request merged**